pkg/controller/mizar/mizar-pod-controller.go

```diff
@@ -34,12 +37,23 @@ import (
 )
 
 const (
-	controllerForMizarPod = "mizar_pod"
+	controllerForMizarPod     = "mizar_pod"
+	defaultNetworkName        = "default"
+	vpcSuffix                 = "-default-network"
```

Comment:
Does not seems read from template file. Can template file be removed?

Reply:
Here is for default Vpc name format, which is {{.Tenant}}-default-network.  So const vpcSuffix is defined as "-default-network". The default vpc name is no need to read from template file.

The template for default vpc below can not be removed because it also contains "ip", "prefix", "dividers".

{
   "apiVersion": "mizar.com/v1",
   "kind": "Vpc",
   "metadata": {
      "name": "{{.Tenant}}-default-network"
   },
   "spec": {
      "ip": "21.0.0.0",
      "prefix": "16",
      "dividers": 1,
      "status": "Init"
   }
}